zulipterminal/ui.py

```diff
@@ -314,6 +319,25 @@ def mouse_event(
 
         return super().mouse_event(size, event, button, col, row, focus)
 
+    def render(self, size: urwid_Box, focus: bool) -> Any:
+        maxcols, maxrows = size
+
+        if not self.autohide:
+            if maxcols < MINNORMALWIDTH and self.mode != "small":
+                self.mode = "small"
+                if self.body.focus_position == 2:
+                    self.show_left_panel(visible=False)
+                else:
+                    self.show_right_panel(visible=False)
```

Comment:
This works well, but perhaps if we're in the central panel we should close both sides?

Reply:
I think only hiding 1 panel is better? Or having both disappear might be confusing?

Reply:
This will perhaps depend on the tab feature.

Reply:
Yes, and also not hiding both would confuse the user as the left column stays but the right column appears and disappears on right then left keypress. I have made both disappear.